packages/mobile/e2e/src/utils/retries.js

```diff
@@ -18,8 +18,10 @@ export const launchApp = async (
         throw error
       }
     },
-    { retries: 5, delay: 10000, timeout: 300000 }
-  )
+    { retries: 5, delay: 10 * 1000, timeout: 30 * 10000 }
+  ).then(async () => {
+    await device.setURLBlacklist(['.*blockchain-api-dot-celo-mobile-alfajores.*'])
```

Comment:
Nicee, was really looking forward to this! 🙌  I don't think it was working properly before, right?

Reply:
It was not working correctly before. I had to do this wrapping to make sure it only fires once the retries is successful. This was what I believe significantly reduced the test times.